e2e/mock-api/src/app/routes.auth.js

```diff
@@ -348,6 +348,7 @@ export default function (app) {
         url.searchParams.set('state', req.query.state);
         res.redirect(url);
       } else if (req.headers.accept.includes('html')) {
+        url.searchParams.set('auth', true);
```

Comment:
Another code comment tying this to the other small change would probably be helpful to our future selves.